<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户中心</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(10, 10, 10);
font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>186****1152</string>
   </property>
  </widget>
  <widget class="QPushButton" name="memberButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>65</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>会员</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="honorButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>65</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>摩范分良好  &gt;</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>451</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mobiLayout">
    <item>
     <widget class="QPushButton" name="mobiButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="mobiSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="myMobi">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>我的摩币</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mobiLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(50, 50, 50);
font: 12pt &quot;新宋体&quot;;</string>
      </property>
      <property name="text">
       <string>24摩币</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>myMobi</zorder>
   <zorder>mobiSpacer</zorder>
   <zorder>mobiButton</zorder>
   <zorder>mobiLabel</zorder>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>670</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="configLayout">
    <item>
     <widget class="QPushButton" name="settingButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="logOffButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="userButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>451</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="remainderLayout">
    <item>
     <widget class="QPushButton" name="remainderButton">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="remainderSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="myRemainder">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>我的钱包</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="remainderLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(50, 50, 50);
font: 12pt &quot;新宋体&quot;;</string>
      </property>
      <property name="text">
       <string>余额24元</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>451</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="tripLayout">
    <item>
     <widget class="QPushButton" name="tripButton">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="tripSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>30</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="myTrip">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>我的行程</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tripLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(50, 50, 50);
font: 12pt &quot;新宋体&quot;;</string>
      </property>
      <property name="text">
       <string>里程数20公里</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>myTrip</zorder>
   <zorder>tripButton</zorder>
   <zorder>tripLabel</zorder>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>accountLabel</zorder>
  <zorder>memberButton</zorder>
  <zorder>honorButton</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>horizontalLayoutWidget_3</zorder>
  <zorder>horizontalLayoutWidget_4</zorder>
  <zorder>userButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
